src/activities/parimal/Parimal.qml

```diff
@@ -0,0 +1,134 @@
+/* GCompris - parimal.qml
+ *
+ * Copyright (C) 2018 PARIMAL PRASOON <parimalprasoon7@gmail.com>
+ *
+ * Authors:
+ *   <THE GTK VERSION AUTHOR> (GTK+ version)
```

Comment:
if it's not a port, it's not needed else you can replace it with the original author